Chapter7/shapes.erl

```diff
@@ -0,0 +1,19 @@
+-module(shapes).
+-export([perimeter/1, area/1]).
+
+-record(circle, {radius}).
+-record(rectangle, {length, width}).
+-record(triangle, {side1, side2, side3}).
+
+% Exercise 7-4: Records and Shapes
+
+perimeter( #circle{radius=Radius} ) 							-> 2 * math:pi() * Radius;
+perimeter( #rectangle{length=Length, width=Width} ) 			-> (2 * Length) + (2 * Width);
+perimeter( #triangle{side1=Side1, side2=Side2, side3=Side3} ) 	-> Side1 + Side2 + Side3.
+
+
+area( #circle{radius=Radius} ) 								-> math:pow(math:pi() * Radius, 2);
```

Comment:
I've updated the code to follow the coding guidelines, I've also changed the settings on the text editor so any future code will have all tabs automatically replaced with two spaces (So only spaces will be used and the indentation will look nice)

Also on the guidelines page you linked me to I couldn't find any information about if its recommended to align text or not, is the following a good coding practice or bad:

test(Arg, ArgLong) when Arg          <  ArgLong -> true;
test(Arg, ArgLong) when ArgLong  < Arg         -> false;

or should things like that be left as:

test(Arg, ArgLong) when Arg <  ArgLong -> true;
test(Arg, ArgLong) when ArgLong  < Arg -> false;